model/registry/linux.yaml

```diff
@@ -16,3 +16,15 @@ groups:
         stability: experimental
         brief: "The Linux Slab memory state"
         examples: ["reclaimable", "unreclaimable"]
+  # linux.process.* attribute group
+  - id: registry.linux.process
```

Comment:
it seems windows has a similar concept of [Job objects](https://learn.microsoft.com/en-us/windows/win32/procthread/job-objects), see https://thomasvanlaere.com/posts/2021/06/exploring-windows-containers/ for some context. 

The suggestion here is to add cross-platform attribute in `process` namespace. I don't mind it being called `process.cgroup`, `process.group.name` or something else

Reply:
If we think this attribute can be cross-platform, I am also in favor of removing the `linux` keyword. As explained [in https://github.com/open-telemetry/semantic-conventions/issues/1403 ](https://github.com/open-telemetry/semantic-conventions/issues/1403#issuecomment-2368815634), and as a general concept, `process.cgroup` would fit in Bucket 1.

Wdyt @open-telemetry/semconv-system-approvers ?